### ISSUE TITLE
Include a path for meetups to join the Collective

### ISSUE THREAD

**Issue by keywordnew** [cambiecollective/cambiecollective.github.io#33]
IMO, "join us" implies a way for folks to join the Collective, rather than contribute to the org through Github.

<img width="529" alt="Screen Shot 2019-05-01 at 3 19 09 PM" src="https://user-images.githubusercontent.com/10335475/57046193-8b1bfb00-6c24-11e9-8640-b28bc0383c7a.png">

**Comment by Qard** [comment]
True. The website is a bit light on content in general, so perhaps we could elaborate a bit on both joining and contributing. 🤔

**Comment by keywordnew** [comment]
I'm wondering about the first items of value that someone who wanted to contribute could do through the Collective to help all the meetups in it.

So far, one big one I've found is building and maintaining a speaker CFP form. Where folks could propose a talk and check all the meetups that they would be willing to speak at.

Would love more suggestions! We could make a bunch of good first issues with them :)
